manifest.yaml

```diff
@@ -0,0 +1,3 @@
+# Default RHEL version used to build RHCOS
```

Comment:
Are we OK with the concept of a "default manifest"? Might be cleaner to just rip off the bandaid and require users/pipelines being more specific going forward. I understand wanting to minimize disruption though. Though... we did fork already for 4.11, so seems OK if this breaks things and we fix them.

Reply:
Due to https://github.com/coreos/coreos-assembler/commit/480c2396977921ffd6cbc4ea44b3fcd2c470c954 we currently need a default. We could however change cosa to not require that.